code/headers/gcode_generator.hpp

```diff
@@ -28,7 +61,7 @@ namespace r2d2::robot_arm {
          * @param char array
          * @return char* to string
          * */
-        char *int_to_string(int axis, char *string) const {
+        char *int_to_string(int axis, char *string) {
```

Comment:
Same as above. Het is weird om een char pointer terug te geven als je de char pointer bewerkt.